### ISSUE TITLE
Fix improve accessibility score

### ISSUE THREAD

**Issue by kyaaron** [kyaaron/Level-Ground-Company-Site#3]
Issue: Lighthouse currently scores the accessibility on the LG site as a 77 due to lack of alt text on images, background differences not having a high enough contrast, and heading elements are not in the correct descending order on the site.

Update the headers and alt-text on images to improve the score. The background contrast will remain as is.

**Comment by kyaaron** [comment]
While working on accessibility, also add a <meta> page description for SEO. The alt text will also help both SEO and accessibility scores for the site.
